### PR TITLE
Added parameter to navigation

### PR DESCRIPTION
**Pull request by krisadamstv**

### CODE REVIEW

File: RokredUI/Services/INavigationService.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace RokredUI.Services
 {
     public interface INavigationService
     {
-        IObservable<Unit> Push<T>(T viewModel) where T : class, IViewModel;
+        IObservable<Unit> Push<T>(object parameter) where T : class, IViewModel;
```
Comment (krisadamstv) [inline RokredUI/Services/INavigationService.cs:9]: Could be T2 and IParameter or whatever. But do you see what I mean about the parameter Edgar?

Comment (edgarfgp) [reply RokredUI/Services/INavigationService.cs:9]: How are we going to the parameter when we get to the new ViewModel.

Comment (edgarfgp) [reply RokredUI/Services/INavigationService.cs:9]: We will need to have some method that will intercept the life cycle and get the values ?

Comment (krisadamstv) [reply RokredUI/Services/INavigationService.cs:9]: This is a working example ;) 
Check the ViewModel1 going to ViewModel2 on my stuff I checked in

Comment (edgarfgp) [reply RokredUI/Services/INavigationService.cs:9]: I would like to talk to you tomorrow about this, I find out a better way

**Pull request closed without merging**